raiden/storage/serialization/cache.py

```diff
@@ -0,0 +1,67 @@
+from typing import Any, Callable, Dict
```

Comment:
What is the purpose of this module and why it was added?

Reply:
Deleted in next commit. This was part of a major change that i tryied to do, but was not worth it.